src/start.sh

```diff
@@ -58,6 +58,14 @@ start() {
     fix_capabilities
     sh /opt/pihole/pihole-FTL-prestart.sh
 
+    # Get the FTL log file path from the config
+    FTLlogFile=$(getFTLConfigValue files.log.ftl)
+
+    # Get the FTL log file line count before FTL starts
+    local startFrom
+    startFrom=$(grep -c "" "${FTLlogFile}")
+    ((++startFrom))
+
```

Comment:
```suggestion
    # Get the FTL log file path from the config
    FTLlogFile=$(getFTLConfigValue files.log.ftl)

    # Get the FTL log file line count before FTL starts
    local startFrom
    startFrom=$(grep -c "" "${FTLlogFile}")
    # Increment by one to start from the first new line in the file
    ((++startFrom))

```

Can we please move this section down a few lines? I'd suggest around line 80, between `CAPSH_PID=$!` and the `(tail...` part. I think it is easier to follow when the declaration is closer to the first usage.

Also some kind of comment (doesn't have to be as suggested) above the incrementing of the `startFrom` variable makes it easier to understand what is happening

Reply:
That's not possible. The length/size of the log file must be determined before FTL starts.

---

I've removed the incrementing, as it's not really needed with the 'pihole-FTL wait-for' approach.